backend/src/routes/v2/model/file/postStartMultipartUpload.ts

```diff
@@ -10,35 +15,61 @@ export const postStartMultipartUploadSchema = z.object({
   body: z.object({
     name: z.string(),
     mime: z.string().optional().default('application/octet-stream'),
-
-    size: z.number(),
+    size: z.number().positive(),
+    tags: coerceArray(z.array(z.string()).optional()),
   }),
 })
 
-interface PresignedChunk {
-  presignedUrl: string
+registerPath({
+  method: 'post',
+  path: '/api/v2/model/{modelId}/files/upload/multipart/start',
+  tags: ['file'],
+  description: 'Start uploading a new multipart file to a model.',
+  schema: postStartMultipartUploadSchema,
+  responses: {
+    200: {
+      description: "The newly created file's chunk details.",
+      content: {
+        'application/json': {
+          schema: z.object({
+            fileId: z.string().openapi({ example: '67cecbffd2a0951d1693b396' }),
+            uploadId: z.string().openapi({ example: '67cecbffd2a0951d1693b396' }),
+            chunks: z.array(z.object({ startByte: z.number(), endByte: z.number() })),
+          }),
+        },
+      },
+    },
+  },
+})
+
+export interface ChunkByteRange {
   startByte: number
   endByte: number
 }
 
 interface PostStartMultipartUpload {
   fileId: string
-  chunks: Array<PresignedChunk>
+  uploadId: string
+  chunks: Array<ChunkByteRange>
 }
 
 export const postStartMultipartUpload = [
+  bodyParser.json(),
   async (req: Request, res: Response<PostStartMultipartUpload>): Promise<void> => {
-    const _ = parse(req, postStartMultipartUploadSchema)
+    req.audit = AuditInfo.CreateFile
+    // Does user have permission to upload a file?
```

Comment:
This comment isn't needed as you're doing an auth check inside `startUploadMultipartFile`